### ISSUE TITLE
Incorrect backend test for LogSoftmax

### ISSUE THREAD

**Issue by kit1980** [onnx/onnx#1210]
The model.onnx at https://github.com/onnx/onnx/tree/master/onnx/backend/test/data/pytorch-converted/test_log_softmax_lastdim has `axis=-1`, which has no meaning in the spec for LogSoftmax.

**Comment by houseroad** [comment]
axis=-1 means the last dimension. We have discussed this long time ago. We align with Numpy, so we decided to support negative axis.

**Comment by kit1980** [comment]
`axis=-1` is not mentioned in the spec for LogSoftmax, and PyTorch has special handling to get rid of negative axis for Softmax, but not for LogSoftmax (plus there are other issues with PyTorch LogSoftmax exporting, see https://github.com/pytorch/pytorch/pull/9576).

**Comment by houseroad** [comment]
The pr is merged.
